src/test/java/gov/healthit/chpl/aqa/stepDefinitions/DpManagementSteps.java

```diff
@@ -0,0 +1,57 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+import static org.testng.Assert.assertTrue;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openqa.selenium.WebDriver;
+
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import gov.healthit.chpl.aqa.pageObjects.DpManagementPage;
+
+/**
+ * Class DpManagementSteps definition.
+ */
+
+public class DpManagementSteps {
+    private WebDriver driver;
+    private static final int TIMEOUT = 30;
```

Comment:
Is TIMEOUT used here? If not, it shouldn't be declared. If it should be used, then I don't see it...